src/sql/workbench/browser/modal/modal.ts

```diff
@@ -412,6 +412,24 @@ export abstract class Modal extends Disposable implements IThemable {
 		});
 	}
 
+	/**
+	* removes the footer button matching the provided label
+	* @param label Label on the button
+	*/
+	protected removeFooterButton(label: string): void {
+		let buttonIndex = this._footerButtons.findIndex(e => {
```

Comment:
Can just use find to return the item directly - slightly more succient

Reply:
using the index to splice